junifer/markers/sphere_aggregation.py

```diff
@@ -0,0 +1,144 @@
+"""Provide class for sphere aggregation."""
+
+# Authors: Federico Raimondo <f.raimondo@fz-juelich.de>
+#          Synchon Mandal <s.mandal@fz-juelich.de>
+# License: AGPL
+
+from typing import Dict, List, Optional
+
+from nilearn.maskers import NiftiSpheresMasker
+
+from ..api.decorators import register_marker
+from ..data import load_coordinates
+from ..utils import logger, raise_error
+from .base import BaseMarker
+
+
+@register_marker
+class SphereAggregation(BaseMarker):
+    """Class for sphere aggregation.
+
+    Parameters
+    ----------
+    coords: str
+        The name of the coordinates list to use. See junifer.data.coordinates
+    radius: float
+        The radius of the sphere in mm. If None, the signal will be extracted
+        from a single voxel. See :class:`nilearn.maskers.NiftiSpheresMasker`
```

Comment:
```See `nilearn.maskers.NiftiSpheresMasker` ...```

Reply:
Intersphinx requires the :class: prefix to work.